Pod/Classes/Keyboard/Keyboard.swift

```diff
@@ -51,18 +51,6 @@ public class Keyboard {
 
 }
 
-extension UIViewAnimationCurve {
-    func animationOption() -> UIViewAnimationOptions {
-        switch self {
-        case .easeInOut: return UIViewAnimationOptions()
```

Comment:
#? any reason not to keep this code, but do the `<< 16` in the `default` case? I like the idea of keeping things in Documented-Land as much as possible, and while the `<< 16` is documented, I'm not sure the values of the curves are guaranteed to be the same 1–4 values as they are currently.

Reply:
Mm that should work. Was trying to consolidate, but yeah I'll add this back in